src/components/SpdxPicker.js

```diff
@@ -18,6 +19,35 @@ export default class SpdxPicker extends Component {
     promptText: PropTypes.string
   }
 
+  constructor(props) {
+    super(props)
+    this.onKeyPress = this.onKeyPress.bind(this)
+    this.onChange = this.onChange.bind(this)
+    this._typeahead = React.createRef()
+  }
+
+  onKeyPress(event, onChange) {
+    const instance = this._typeahead.current.typeahead.getInstance()
+    const isMenuOpen = instance.state.showMenu
+    const resultsExist = instance.state.initialItem != null
+    const enterPressed = event.key === 'Enter'
+    // if user is in mid-selection, don't hijack Enter key
+    // i.e. only fire onChange on Enter if menu closed or no results
+    if (enterPressed && (!isMenuOpen || !resultsExist)) {
+      const { target } = event
+      const { value } = target
+      value && this.onChange(value, onChange)
+    }
```

Comment:
can this validation be updated to support license expressions?

If I enter `MIT OR Apache-2.0` and hit enter, i would like it to work if it is valid

Reply:
It was already in place, but the check for the enter key was avoided by the previous implementation. Now it should be fine.

Reply:
👍 👍